src/v1.rs

```diff
@@ -370,17 +372,18 @@ impl traits::Aggregator for Aggregator {
 
     /// Initialize the Aggregator polynomial
     fn init(&mut self, comms: &HashMap<u32, PolyCommitment>) -> Result<(), AggregatorError> {
+        let threshold = self.threshold.try_into().unwrap();
```

Comment:
`usize` is not always bigger than `u32`, so this can fail. But as @jferrant  discovered last year, if`as usize` casts fail they do so silently 😳

So now we always do try/unwrap.

Reply:
Oh interesting. Well in that case why would you panic here? Why not just bubble up the error and have the caller handler it if they want?

---
Side note. I thought that on "regular" machines, like machines where the CPU instruction set defaults to 64bit or 32bit "words" (I forget the proper term), that `usize` will be at least 32 bits. Also, I thought that the `as` casts just reinterprets the bits, so a failure "just" means that the number isn't what you thought it was. Is that what you mean by silent failure?

Reply:
I think @jferrant ’s logic was basically: this will almost certainly never fail. But if it does, it means something terribly wrong has happened, so let’s make sure we notice it by panicking. 

And yes, that’s what we mean by silent failure here.

Reply:
The reason not to bubble up is the sheer number of these present after we switched to using u32/u64. Previously we used usize over the network which was super sketchy.

Reply:
Actually the number is not too large; only 26 sites, some of which are in functions that already return Result, and other in test code.  I'll add an issue to take care of the low hanging fruit.

Reply:
Tracking this work in #89

Reply:
PR is up, though it should go in after this one since it will need to handle the unwraps here: #91